src/main/scala/nl/knaw/dans/api/sword2/CollectionDepositManagerImpl.scala

```diff
@@ -75,15 +88,26 @@ class CollectionDepositManagerImpl extends CollectionDepositManager {
       case t: Throwable => Failure(new SwordError("http://purl.org/net/sword/error/ErrorBadRequest", t))
     }
 
-  private def handleDeposit(id: String, auth: AuthCredentials, deposit: Deposit): Try[Unit] =
+  private def handleDepositAsync(id: String, auth: AuthCredentials, deposit: Deposit): Try[Unit] = Try {
```

Comment:
Maybe call this `finalizeDepositAsync`, in keeping with the terminology in the readme.